ipaserver/plugins/vault.py

```diff
@@ -42,9 +42,10 @@
     # pylint: disable=no-member
     try:
         # pki >= 10.4.0
-        from pki.crypto import DES_EDE3_CBC_OID
+        from pki.crypto import DES_EDE3_CBC_OID, AES_128_CBC_OID
     except ImportError:
         DES_EDE3_CBC_OID = pki.key.KeyClient.DES_EDE3_CBC_OID
+        AES_128_CBC_OID = None
```

Comment:
I think this could be dangerous. If DES_EDE3_CBC_OID ever fails import it will disable AES.

Reply:
This was just experimental pathc. IPA 4.8 requires Dogtag with these constants always defined. I have simplified the code.